### PR TITLE
Adding a default tabbable element to fix accessibility issue bug-8860

### PR DESCRIPTION
**Pull request by ankityadav4**
## New Behavior
Adding a default tabbable element to fix accessibility issue bug-8860

## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/35c85db9215be83c0e9c71250648d1587eb5f97f'>35c85db9215be83c0e9c71250648d1587eb5f97f</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx [hunk 1]
```diff
@@ -463,7 +465,14 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
         ref={(rootElem: HTMLDivElement) => (this.chartContainer = rootElem)}
         onMouseLeave={this._onChartLeave}
       >
-        <FocusZone direction={focusDirection} className={this._classNames.chartWrapper} {...svgFocusZoneProps}>
+        {!this._isFirstRender && <div id={this.idForDefaultTabbableElement} />}
+        <FocusZone
+          direction={focusDirection}
+          className={this._classNames.chartWrapper}
+          defaultTabbableElement={`#${this.idForDefaultTabbableElement}`}
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx:472]: Can we set defaultTabbableElement as null in second render cycle?

Comment (ankityadav4) [reply packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx:472]: I tried this , but this does not for a few charts , also I saw Focus Zone code the logic to updateTabIndexes gets triggered only when there is a defaultTabbableElement and then it is no longer inside FocusZone element([code](https://github.com/ankityadav4/fluentui/blob/4b802ba3745dde01c68224329c1414f077faa1db/packages/react-focus/src/components/FocusZone/FocusZone.tsx#L244))

File: packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx [hunk 2]
```diff
@@ -463,7 +465,14 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
...
+          {...svgFocusZoneProps}
+        >
+          {this._isFirstRender && <div id={this.idForDefaultTabbableElement} />}
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx:475]: We need this only when optimizeData flag is set to true

Comment (ankityadav4) [reply packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx:495]: Without flag ,it is fixing this for Bar charts.

**Review by AtishayMsft — APPROVED** [review]

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**